Lib/datetime.py

```diff
@@ -1210,6 +1210,34 @@ def __reduce__(self):
         else:
             return (self.__class__, args, state)
 
+
+class IsoCalendarDate(tuple):
+
+    def __new__(cls, year, week, weekday):
+        return super().__new__(cls, (year, week, weekday))
+
+    @property
+    def year(self):
+        return self[0]
+
+    @property
+    def week(self):
+        return self[1]
+
+    @property
+    def weekday(self):
+        return self[2]
+
+    def __reduce__(self):
+        return (tuple, (tuple(self),))
```

Comment:
Is it deliberate to deserialize it as a regular tuple? If yes, please add a comment to explain the rationale.